syntax = "proto3";

package amocrm_connect_service;

service AmocrmConnectService {
  rpc TryConnect (AmocrmConnectRequest) returns (AmocrmConnectResponse);
}

message AmocrmConnectRequest {
  string login = 1;
  string password = 2;
  string host = 3;
}


message Data {
  string message = 1;
}

message AmocrmConnectResponse {
  bool answer = 1;
  bool success = 2;
  Data data = 3;
  float execution = 4;
}
